tests/test-infrastructure/docker-compose-govtool.yml

```diff
@@ -68,3 +68,15 @@ services:
       placement:
         constraints:
           - node.labels.govtool==true
+  storybook:
+    image: govtool/storybook:${GOVTOOL_TAG}
+    build:
+      context: ../../govtool/frontend
+      dockerfile: Dockerfile.story
+      args:
+        NPMRC_TOKEN: ${NPMRC_TOKEN}
+    environments:
+      VIRTUAL_HOST: https://${BASE_DOMAIN}-storybook
```

Comment:
@NabinKawan Could you verify this?

Reply:
this is renamed to `storybook-${BASE_DOMAIN}`